messages/device-added-notification.json

```diff
@@ -38,7 +38,419 @@
+                            "description": "The href of the link"
+                          },
+                          "rel": {
+                            "type": "string",
+                            "title": "Link rel",
+                            "description": "The type of the relationship"
+                          },
+                          "mediaType": {
+                            "type": "string",
+                            "title": "Link media type",
+                            "description": "The media type of the link"
+                          }
+                        }
+                      }
+                    },
+                    "type": {
+                      "type": "string",
+                      "title": "Property value type",
+                      "description": "The type of the property value"
+                    },
+                    "unit": {
+                      "type": "string",
+                      "title": "Property unit",
+                      "description": "The unit of the property"
+                    },
+                    "minimum": {
+                      "type": "number",
+                      "title": "Property minimum value",
+                      "description": "The minimum value of the property"
+                    },
+                    "maximum": {
+                      "type": "number",
+                      "title": "Property maximum value",
+                      "description": "The maximum value of the property"
+                    },
+                    "multipleOf": {
+                      "type": "number",
+                      "title": "Property value precision",
+                      "description": "The precision of the value"
+                    },
+                    "enum": {
+                      "type": "array",
+                      "items": {
+                        "type": "string",
+                        "title": "Property values enum",
+                        "description": "The possible values of the property"
+                      }
+                    },
+                    "readOnly": {
+                      "type": "boolean",
+                      "title": "Property read-only",
+                      "description": "If the property is read-only"
+                    },
+                    "visible": {
+                      "type": "boolean",
+                      "title": "Property visible",
+                      "description": "If the property is visible"
+                    }
+                  }
+                }
+              }
+            },
+            "actions": {
+              "type": "object",
+              "patternProperties": {
+                ".*": {
+                  "type": "object",
+                  "title": "Action",
+                  "description": "Description of the Action",
+                  "properties": {
+                    "title": {
```

Comment:
What about `name` and `@type`?

Reply:
They weren't listed [here](https://iot.mozilla.org/wot/#action-object).
Also, the `name` is already the key for the object, so this would be redundant.

Reply:
Just saw that `@type` is in the example.
I thought the listing is `exhaustive`.

Reply:
Ok, maybe `name` isn't useful, but `@type` definitely is. We use it in the gateway.

Reply:
Added